src/mrpro/data/_kdata/KDataRemoveOsMixin.py

```diff
@@ -61,7 +61,7 @@ def crop_readout(data_to_crop: torch.Tensor):
         ks = [self.traj.kz, self.traj.ky, self.traj.kx]
         # only cropped ks that are not broadcasted/singleton along k0
         cropped_ks = [crop_readout(k) if k.shape[-1] > 1 else k.clone() for k in ks]
-        cropped_traj = KTrajectory(*cropped_ks)
+        cropped_traj = KTrajectory(cropped_ks[0], cropped_ks[1], cropped_ks[2])
```

Comment:
actually not that bad IMO

Reply:
yeah, this allows mypy to properly check that...